### DISCUSSION TITLE
steamtinkerlaunch and Cheat Engine

### DISCUSSION THREAD

**Discussion by zany130** [ScoopInstaller/Extras#8699] [General]
we were looking at how we can automatically install newer versions of cheat engine (we were using innoextract but that no longer seems to be an option with newer versions?) [here](https://github.com/frostworx/steamtinkerlaunch/issues/497)  and I found this  https://raw.githubusercontent.com/ScoopInstaller/Binary/master/cheat-engine/CheatEngine74.7z and was wondering if you guys mind if we use that. until now we have been relying on a waybackmachine version of 7.2 that didn't have all the adware bundled

**Comment by rashil2000** [comment]
The binaries hosted on the ScoopInstaller/Binary repo are under Unlicense, as long as your usage complies with that license you're free to use any of them
